bundle/README.md

```diff
@@ -256,6 +256,13 @@ To install Prometheus you will need to follow this steps:
         ```sh
         $ oc get route kiali -o jsonpath='{.spec.host}' -n istio-system
         ```
+
+## Type-Safe Helm values API
```

Comment:
This title is so technical jargon that even I don't know what it means :)

Can we make it more human-friendly, e.g. `Input validation` or similar?

Reply:
@jwendell I've only heard it called Type-Safe Helm values, so I thought this was something the customer would know. Thank you for bringing this up for discussion. I'm definitely happy to change it to make it more helpful or usable for our customers. How about `Operator input validation`?

Reply:
cc @longmuir @bmangoen @luksa

Reply:
Yeah, I think there was an original intent (hope?) of type validations (ie validate if there's a string where there should be a number or vice versa)... but as I understand it, that wasn't feasible (it was a nice to have)... so we implemented a validation that would report an error if an unknown field was added - for example, if the user made a typo or tried to configure something that doesn't exist. We should have updated the related Jira issues to correct that direction when the pivot was made to better capture what was being added. Next best thing, let's try to describe it here :). Marko's demo was helpful today with his "foo: bar" example of a bogus field.

As for what to title the section... maybe "Configuration Validations", "Values Validations" or even just "Validations". I think it's also important to describe what the 'rawValues' field was in relation to the validations applied to "values".

Reply:
For what it's worth... starting to write about this in the blog, I am leaning toward "Validation of Configuration".

Reply:
I've updated the PR and changed the title to Validating the configuration, just to try and follow Red Hat's style guidelines.